onyo/lib/onyo.py

```diff
@@ -38,47 +38,104 @@ def __init__(self, path: Union[Path, str], init: bool = False) -> None:
 
     @property
     def assets(self) -> set[Path]:  # pyre-ignore[11]
+        """
+        A `set` containing the `Path`s to all assets of an Onyo repository.
+
+        If the property is not set, load the assets of the repository and save
```

Comment:
IMO, this should be removed. It's purely a private concern, and a consumer does not care.